synapse/lib/config.py

```diff
@@ -161,14 +161,19 @@ def getArgParseArgs(self):
             if conf.get('hideconf'):
                 continue
 
+            typename = conf.get('type')
```

Comment:
inline help for future self - only allow single-typed values to have command line arguments